docs/reference/commandline/service_create.md

```diff
@@ -678,6 +678,25 @@ appends a new placement preference after all existing placement preferences.
 `--placement-pref-rm` removes an existing placement preference that matches the
 argument.
 
+### Specify the minimum memory needed for a service (--reserve-memory)
+
+If your service needs a minimum amount of memory in order to run correctly,
+you can use `--reserve-memory` to specify that the service should reserve a
+given amount of memory on workers. The following example reserves 4GB of
+memory for the service.
```

Comment:
Let's explain that this is relevant to the scheduling decisions. For example, the managers won't schedule a set of containers on a single node whose combined reservations exceed the memory available on that node.